README.md

```diff
@@ -228,6 +228,70 @@ try {
 ```
 Note that in the closure-based example above, exceptions are intercepted automatically.
 
+### Oversampling Mitigation
+Oversampling mitigation allows you to ignore a parent segment/subsegment's sampled flag and instead set it to false.
```

Comment:
I think this sentence still needs some rewording - it is unclear what "it" is referring to here